### DISCUSSION TITLE
How about a change log?

### DISCUSSION THREAD

**Discussion by stevecj** [carltongibson/django-filter#1652] [Ideas]
Trying to upgrade our Python packages to newer versions, I had a test failure and narrowed it down to upgrading from django-filter 21.1 to anything 22.x or newer. The failures are in assertions and are not exceptions, so it is not obvious what kind of problem I am having.

The first thing I did was look everywhere for some kind of change log to see if there are any compatibility-breaking changes between 21.x and 22.x, but I found nothing.

A change log would be very helpful for this kind of thing.

**Comment by jeremystretch** [comment]
Is [this](https://github.com/carltongibson/django-filter/blob/main/CHANGES.rst) what you're looking for?

**Reply by stevecj** [reply]
I am clearly blind. Thanks. 😄

**Comment by carltongibson** [comment]
It's probably the removal of the renamed attributes from the 2.0 update (there's a migration guide in the docs). Those were raising deprecation warnings for a number of years, but the fallbacks were finally removed then.
